definitions/organization_user_group_list.yml

```diff
@@ -0,0 +1,36 @@
+# yaml-language-server: $schema=.schema.yml
+beta: true
+location: internal/plugin/service/organization/usergrouplist
+datasource:
+  description: List all user groups within an organization
+idAttribute:
+  fields:
+    - organization_id
+operations:
+  UserGroupsList: read
+schema:
+  user_groups:
+    description: User Groups
+    items:
+      properties:
```

Comment:
You can avoid specifying the whole list of properties if they are suitable in the API response. However, I dunno if we should expose all information, and does it have any benefits, e.g. `create_time`, `update_time`